packages/calling-component-bindings/src/participantListSelector.ts

```diff
@@ -30,14 +30,11 @@ const convertRemoteParticipantsToParticipantListParticipants = (
           const isScreenSharing = Object.values(participant.videoStreams).some(
             (videoStream) => videoStream.mediaStreamType === 'ScreenSharing' && videoStream.isAvailable
           );
-          return memoizeFn(
-            toFlatCommunicationIdentifier(participant.identifier),
-            participant.displayName,
-            participant.state,
-            participant.isMuted,
+          return memoizeFn({
+            userId: toFlatCommunicationIdentifier(participant.identifier),
             isScreenSharing,
-            participant.isSpeaking
-          );
+            ...participant
```

Comment:
this adds properties (like videoStreams) which I don't think we want here?

Reply:
Removed the spread. Thanks.